<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1437</width>
    <height>906</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Kết quả</string>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>581</width>
        <height>571</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Tên</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Khóa</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Kiểm tra</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Xác suất</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="lbAnhGoc">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>221</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbAnhPhatHien">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>60</y>
        <width>161</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>260</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ảnh gốc</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>260</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ảnh phát hiện</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" rowspan="2" colspan="2">
       <widget class="QLabel" name="lbAnh">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QToolButton" name="btnDialog">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="btnFace">
           <property name="text">
            <string>Nhận dạng qua ảnh</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="btnFaceCamera">
           <property name="text">
            <string>Nhận dạng qua Camera</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="btnChup">
           <property name="text">
            <string>Chụp ảnh</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QDateTimeEdit" name="dtNow"/>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btnTiny">
           <property name="text">
            <string>Ảnh khuôn mặt nhỏ</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btnCheckTiny">
           <property name="text">
            <string>Rà Soát học viên</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Thống kê</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Tổng số có mặt:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Vắng mặt: </string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Unknow: </string>
         </property>
        </widget>
        <widget class="QLabel" name="lbCoMat">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>60</y>
           <width>55</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>.....</string>
         </property>
        </widget>
        <widget class="QLabel" name="lbVangMat">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>100</y>
           <width>55</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>.....</string>
         </property>
        </widget>
        <widget class="QLabel" name="lbUnknow">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>140</y>
           <width>55</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>.....</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnReset">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>240</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Mặt quá bé:</string>
         </property>
        </widget>
        <widget class="QLabel" name="lbFaceClose">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>190</y>
           <width>55</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>.....</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
